Main/Peripherals/include/Peripherals/ThrustController.h

```diff
@@ -37,8 +39,11 @@ class ThrustController
 
     FloatPair forwardTrim;
     FloatPair diveTrim;
+    FloatPair diveOffset;
 
     FloatPair getReciprocalValues(float value);
+    float getSafeOffset(float a, float b);
+    void zeroPowerHelper(float &a, float &b);
```

Comment:
Passing by referencing is an okay practice in certain situations in C. But it should be avoided completely in C++ or any other OO language. A more functional and readable approach would be to return new values as a FloatPair rather than modifying the existing values.